node/clients/checkout.ts

```diff
@@ -17,16 +17,16 @@ export class Checkout extends JanusClient {
     })
   }
 
-  private getChannelQueryString = () => {
+  private getChannelQueryString = (tradePolicy?: string) => {
     const { segment } = this.context as CustomIOContext
     const channel = segment && segment.channel
-    const queryString = channel ? `?sc=${channel}` : ''
+    const queryString = channel ? `?sc=${tradePolicy || channel}` : ''
     return queryString
   }
 
   public simulation = (simulation: SimulationPayload) =>
     this.post<OrderForm>(
-      this.routes.simulation(this.getChannelQueryString()),
+      this.routes.simulation(this.getChannelQueryString(simulation.tradePolicy)),
       simulation,
```

Comment:
does this route expect to receive simulation with the `tradePolicy` prop? if not, we need to remove it before sending `simulation` here

Reply:
the simulation route? if so, yes! :D

Reply:
hmm i don't think so. I believe that the `tradepolicy` should be passed on the item or by query string, can you remove this information from the `simulation` object before sending?